hazelcast/src/main/java/com/hazelcast/internal/nearcache/impl/store/AbstractNearCacheRecordStore.java

```diff
@@ -334,23 +334,29 @@ public boolean remove(K key) {
             record = removeRecord(key);
             if (record != null && record.getRecordState() == READ_PERMITTED) {
                 removed = true;
+                nearCacheStats.incrementInvalidations();
                 nearCacheStats.decrementOwnedEntryCount();
             }
             onRemove(key, record, removed);
             return record != null;
         } catch (Throwable error) {
             onRemoveError(key, record, removed, error);
             throw rethrow(error);
+        } finally {
+            nearCacheStats.incrementReceivedInvalidations();
```

Comment:
as an addition to the above [comment](https://github.com/hazelcast/hazelcast/pull/12960/files#r184679689) we can call this `incrementInvalidations` (seeing `received` in the name confuses me)